.github/workflows/integration-build-product.yml

```diff
@@ -774,8 +798,424 @@ jobs:
+            pushd "${tests_dir}"
+            rm -rf venv
+            python3 -m venv venv
+            . venv/bin/activate
+            python3 -m pip install --upgrade pip
+            pip install -i https://artifacts.cloud.mov.ai/repository/pypi-integration/simple --extra-index-url https://pypi.org/simple -r requirements.txt
+            deactivate
+            popd
+
+          '
+          rm -f /tmp/target_dir.txt /tmp/version.txt /tmp/repo_name.txt /tmp/test_set.txt
+          scp -r robot:/tmp/*.txt /tmp/
+
+          echo "target_dir=$(cat /tmp/target_dir.txt)" >> $GITHUB_OUTPUT
+          echo "version=$(cat /tmp/version.txt)" >> $GITHUB_OUTPUT
+          echo "test_set=$(cat /tmp/test_set.txt)" >> $GITHUB_OUTPUT
+
+      - name: Remote - Install
+        if: ${{ inputs.with_simulation_tests }}
+        id: install
+        shell: bash
+        run: |
+          ssh robot '
+          set -e
+          cd ${{ env.REMOTE_WORKSPACE }}
+
+          for robot in $(movai-cli robots list); do
+            movai-cli robots stop $robot
+            sleep 5
+            movai-cli robots remove $robot
+          done || true
+
+          mkdir -p artifacts
+          cp *.json artifacts/
+          CONFIG_FILE_NAME="standalone-${{ inputs.product_name }}-simulator-noetic.json"
+          export PATH="$HOME/.local/bin:$PATH"
+          integration-pipeline get_json_value --file $CONFIG_FILE_NAME.ci --key services_version --output_file movai_service_version
+          integration-pipeline get_json_value --file $CONFIG_FILE_NAME.ci --key quickstart_version --output_file quickstart_version
+
+          export USERSPACE_FOLDER_PATH="$(pwd)/userspace"
+          export PUBLIC_IP=$(hostname -I | awk "{print $1}")
+          export SIMULATION_ID="CI"
+          export DISPLAY=":1"
+          rm -rf userspace
+
+          wget https://movai-scripts.s3.amazonaws.com/QuickStart_$(cat quickstart_version).bash
+          chmod +x ./QuickStart_$(cat quickstart_version).bash
+          ./QuickStart_$(cat quickstart_version).bash --apps $(cat movai_service_version) $CONFIG_FILE_NAME
+          MOVAI_USER="ci"
+          MOVAI_PWD="4Iva6UHAQq9DGITj"
+          for robot in $(movai-cli robots list); do
+            movai-cli robots user "$robot" "$MOVAI_USER" "$MOVAI_PWD"
+          done
+
+          execution_status=$?
+          exit $execution_status
+          rm movai_service_version
+          '
+          echo "movai_user=ci" >> $GITHUB_OUTPUT
+          echo "movai_pwd=4Iva6UHAQq9DGITj" >> $GITHUB_OUTPUT
+
+      - name: Remote - Simulation Tests
+        timeout-minutes: 30
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        run: |
+          ssh robot '
+            set -e
+            cd ${{ env.REMOTE_WORKSPACE }}/${{ steps.simulator_tests_setup.outputs.target_dir }}
+
+            # install test dependencies on host
+            sudo apt install -y --allow-downgrades python3-rosnode python3-rosparam python3-rostopic
```

Comment:
those packages should come from a file listing requirements

Reply:
Not against, but these packages are a dependency of the test infrastructure, not the test it self. So I'm not sure if it makes sense

Reply:
ok, once the tests are packaged we can review this sections